schemas/function.yaml

```diff
@@ -43,6 +43,14 @@ $defs:
       description:
         type: string
         description: Describes the functionality provided by the function.
+      important_notes:
```

Comment:
I don't think a new attribute is needed. You should change the notes attribute by giving it more properties other than description like "type" so we can specify the "importance" or color of the note (like warning).